src/Foundation/Commerce/tests/Services/Order/OrderServiceTests.cs

```diff
@@ -0,0 +1,296 @@
+﻿//    Copyright 2020 EPAM Systems, Inc.
+// 
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+// 
+//      http://www.apache.org/licenses/LICENSE-2.0
+// 
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+namespace Wooli.Foundation.Commerce.Tests.Services.Order
+{
+    using System;
+    using System.Collections.Generic;
+
+    using Commerce.Mappers.Order;
+    using Commerce.Services.Order;
+
+    using Connect.Context;
+    using Connect.Managers;
+
+    using Context;
+
+    using NSubstitute;
+
+    using Ploeh.AutoFixture;
+
+    using Sitecore.Commerce.Entities.Carts;
+    using Sitecore.Commerce.Entities.Orders;
+    using Sitecore.Commerce.Services;
+    using Sitecore.Commerce.Services.Carts;
+    using Sitecore.Commerce.Services.Orders;
+
+    using Xunit;
+
+    public class OrderServiceTests
+    {
+        private readonly IOrderMapper mapper;
+        private readonly IOrderManagerV2 orderManager;
+        private readonly ICartManagerV2 cartManager;
+        private readonly IStorefrontContext storefrontContext;
+        private readonly IVisitorContext visitorContext;
+        private readonly OrderService service;
+        private readonly Fixture fixture;
+
+        public OrderServiceTests()
+        {
+            this.mapper = Substitute.For<IOrderMapper>();
+            this.orderManager = Substitute.For<IOrderManagerV2>();
+            this.cartManager = Substitute.For<ICartManagerV2>();
+            this.storefrontContext = Substitute.For<IStorefrontContext>();
+            this.visitorContext = Substitute.For<IVisitorContext>();
+
+            this.service = new OrderService(
+                this.mapper,
+                this.orderManager,
+                this.cartManager,
+                this.storefrontContext,
+                this.visitorContext);
+
+            this.fixture = new Fixture();
+        }
+
+        private (string, GetVisitorOrderResult) InitGetOrder(bool success)
```

Comment:
Private methods usually placed after public

Reply:
But resharper cleanup put it like that

Reply:
It strange, my ReSharper cleanup put it after public